### PR TITLE
statement,slowquery: add request unit part in list

### PR DESCRIPTION
**Pull request by nolouch**
### Statement:

image

Image

### Slow Qurey
image
Image

### CODE REVIEW

File: pkg/apiserver/slowquery/statement_gen.go [hunk 1]
```diff
@@ -52,7 +60,9 @@ func genOrderStmt(tableColumns []string, orderBy string, isDesc bool) (string, e
 	} else {
 		// We have both TiDB 4.x and TiDB 5.x columns listed in the model. Filter out columns that do not exist in current version TiDB schema.
 		fields := lo.Filter(getFieldsAndTags(), func(f Field, _ int) bool {
-			return lo.Contains(tableColumns, f.ColumnName)
+			hasProjection := f.Projection != ""
+			isTableColumnValid := lo.Contains(tableColumns, f.ColumnName)
+			return hasProjection || isTableColumnValid
```
Comment (baurine) [inline pkg/apiserver/slowquery/statement_gen.go:65]: I feel like this part logic should be same as the line 30~38.

**Review by baurine — COMMENTED** [review]
the rest lgtm

**Comment by nolouch** [comment]
ptal @mornyx @baurine

**Review by baurine — APPROVED** [review]

File: pkg/apiserver/slowquery/statement_gen.go [hunk 2]
```diff
@@ -25,9 +27,15 @@ func genSelectStmt(tableColumns []string, reqJSONColumns []string) (string, erro
 
 	// We have both TiDB 4.x and TiDB 5.x columns listed in the model. Filter out columns that do not exist in current version TiDB schema.
 	fields = lo.Filter(fields, func(f Field, _ int) bool {
-		hasProjection := f.Projection != ""
```
Comment (mornyx) [inline pkg/apiserver/slowquery/statement_gen.go:28]: Sorry for my late reply.. Why is `hasProjection` no longer needed?

Comment (nolouch) [reply pkg/apiserver/slowquery/statement_gen.go:28]: I use `related` to replace it, the same as in the statement.

**Pull request merged**